Library/Contracts.ts

```diff
@@ -10,6 +10,7 @@ export module Contracts {
         SQL = 0,
         Http = 1,
         Other = 2,
+        ApplicationInsights = 3,
```

Comment:
Hmmm ... yes ... I think we will need some help from the AI team here. 
For .net it used to be the same. But the public API has since been updated. I am not sure how they handle breaking the public API. 

Here's is the current state of the .net API for reference: 
https://github.com/Microsoft/ApplicationInsights-dotnet/blob/master/src/Core/Managed/Shared/TelemetryClient.cs#L262-L305

Anyhow, we should raise the discrepancy with them. If you don't want to get into removing the 'DependencyKind' contract. I suggest for the new 'Application Insights' dependency type, send 
DependencyKind = Http and DependencyTypeName= 'Application Insights'. 

The ingestion code is well equipped to ignore the Kind attribute and use the DependencyTypeName instead.